references/classification/utils.py

```diff
@@ -409,3 +409,21 @@ def reduce_across_processes(val):
     dist.barrier()
     dist.all_reduce(t)
     return t
+
+
+try:
```

Comment:
This section is borrowed from ClassyVision:
https://github.com/facebookresearch/ClassyVision/blob/main/classy_vision/generic/util.py#:~:text=parameters%20in%20the-,optimizer,-param_groups.

Reply:
Indeed we switched to `torch.cuda.amp` recently. See #4547